_posts/en/newsletters/2020-05-20-newsletter.md

```diff
@@ -0,0 +1,167 @@
+    and efficiently.
+
+    Commenting on the protocol, Lloyd Fournier [noted][fournier
+    elegance] the "elegance" of a simplified version of the protocol
+    that uses three transactions.  Dmitry Petukhov [posted][petukhov
+    tla+] about a [specification][sas tla+ spec] he'd written for the
+    protocol in the [TLA<sup>+</sup> formal specification language][tla+
+    lang], helping to verify the correctness of the protocol (to a
+    point---as Petukhov notes, there's no model for Bitcoin Core's
+    transaction relay policy, so its effect on the protocol can't
+    currently be checked automatically).
+
+## Changes to services and client software
+
+*In this monthly feature, we highlight interesting updates to Bitcoin
+wallets and services.*
+
+FIXME:bitschmidty
+
+## Releases and release candidates
+
+FIXME:harding to update Tuesday afternoon with latest releases and RCs
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [Bitcoin Core 0.20.0rc2][bitcoin core 0.20.0] is the newest release
+  candidate for the next major version of Bitcoin Core.  It
+  [contains][Bitcoin Core #18973] several bug fixes and improvements
+  since the first release candidate.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1 repo],
+[Bitcoin Improvement Proposals (BIPs)][bips repo], and [Lightning
+BOLTs][bolts repo].*
+
+*Note: the commits to Bitcoin Core mentioned below apply to its master
+development branch and so those changes will likely not be released
+until version 0.21, about six months after the release of the upcoming
+version 0.20.*
+
+- [Bitcoin Core #18877][] Serve cfcheckpt requests FIXME:jnewbery
+
+- [Bitcoin Core #18894][] fixes a UI bug that affected people who
+  simultaneously used multi-wallet mode in the GUI and manual coin
+  control.  The bug was described as a [known issue][coin control bug]
+  in the Bitcoin Core 0.18 release notes.  This is included in the
+  second release candidate for Bitcoin Core 0.20 linked in the
+  preceding section.
+
+- [Bitcoin Core #18962][] updates how Bitcoin responds to receiving a
+  P2P protocol `inv` message announcing multiple new blocks.
+  Previously, Bitcoin Core would reply with a `getheaders` message
+  for each block listed in the `inv`.  After the merge of this PR, it
+  will only request the header of the last block announced in the `inv`.
+  There's no need[^getheaders-details] to send more than one `getheaders`
+  request and Bitcoin Core doesn't send more than one block announcement
+  in an `inv` message, so this change should have no effect when Bitcoin
+  Core nodes connect to Bitcoin Core peers; it's only expected
```

Comment:
s/it's/its/ (as this is possessive and not a contraction)

Considering the sentence length, could also break into 2 sentences here with s/; it's/. The/